config/celery.py

```diff
@@ -1,28 +1,66 @@
-"""
-Celery configuration
-"""
-
 from celery import Celery, signals
-from django.conf import settings
+from django.conf import Settings, settings
+from kombu import Queue
+from pydantic_settings import BaseSettings, SettingsConfigDict
+
+
+class CelerySettings(BaseSettings):
+    model_config = SettingsConfigDict(env_prefix="OSIDB_CELERY_")
+
+    enable_fifo: bool = False
+    fifo_pool_size: int = 2
```

Comment:
I think we should add a note somewhere that this number corresponds with the number of celery workers, as each worker listens to only one queue. 
Just to prevent having 2 or more queues and only one worker, as those tasks will never be processed